.github/workflows/build-and-test.yml

```diff
@@ -47,17 +47,31 @@ jobs:
           slack-bot-token: ${{ secrets.SLACK_BOT_TOKEN }}
           channel: github-activities
 
-  metadata-ingestion:
+  metadata-ingestion-general:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
       - uses: actions/setup-python@v2
         with:
-          python-version: "3.8"
+          python-version: "3.6"
       - name: Install dependencies
         run: ./metadata-ingestion/scripts/install_deps.sh
       - name: Run metadata-ingestion tests
-        run: ./gradlew :metadata-ingestion:build :metadata-ingestion:check :metadata-ingestion:testFull
+        run: ./gradlew :metadata-ingestion:build :metadata-ingestion:check
+  metadata-ingestion-by-version:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: ["3.6", "3.9"]
```

Comment:
Why not 3.7 and 3.8?

Reply:
I figured that testing 3.6 (min supported version) and 3.9 (latest released version) would be sufficient. We definitely could add more versions to test if we feel it necessary, but I don't think we're really at risk here.

Reply:
Sounds good!